pkg/disk/cloud.go

```diff
@@ -233,14 +226,13 @@ func attachDisk(ctx context.Context, tenantUserUID, diskID, nodeID string, isSha
 				return "", status.Errorf(codes.Aborted, "NodeStageVolume: failed to attach bdf: %v", err)
 			}
 
-			devicePaths, err = GetDeviceByVolumeID(diskID)
-			deviceName := ""
-			if len(devicePaths) == 0 && bdf != "" {
-				deviceName, err = GetDeviceByBdf(bdf, true)
-			}
-			if err == nil && deviceName != "" {
-				log.Infof("AttachDisk: Successful attach bdf disk %s to node %s device %s by DiskID/Device mapping", diskID, nodeID, deviceName)
-				return deviceName, nil
+			_, err := DefaultDeviceManager.GetRootBlockByVolumeID(diskID)
+			if err != nil && bdf != "" {
```

Comment:
these nest ifs are very confused, try to clarify them.

Reply:
I think this is out of the scope of this PR. I've reverted some changes here